test/patterns/cltools/fileCompile/chplCompile.chpl

```diff
@@ -0,0 +1,54 @@
+
+/*
+A tool written in chapel to compile a chapel program and to demonstrate the use of the 
+spawn module and how it is used to create subprocesses within a chapel program. 
+*/
+module chplCompile {
+
```

Comment:
The usual convention is to indent everything inside of a module. However, since this is a test, you could actually delete this line and the matching curly brace -- the compiler will auto-generate the module name from the filename.